### PR TITLE
#223/feat/add possibility to delete tags

### PR DESCRIPTION
**Pull request by piamanns**
### In this pull request:
- Each tag chip in the tag list now has a corresponding delete button
- Pressing the delete button pops a confirm dialogue. Clicking Confirm deletes the tag, clicking Cancel cancels the deletion
- The translations related to tag handling were moved to the admin locale file

### CODE REVIEW

File: src/app/api/tag/route.ts [hunk 1]
```diff
@@ -38,10 +38,38 @@ export async function POST(request: NextRequest) {
...
   }
 }
+
+export async function DELETE(request: NextRequest) {
+  const { t } = await translator('api');
+  try {
+    const { user } = await getServerAuthSession();
+    if (!isAdmin(user)) {
+      return errorResponse({
+        message: 'Forbidden',
```
Comment (tommivk) [inline src/app/api/tag/route.ts:55]: Missing translation

File: src/components/TagList/TagChip.tsx [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+      <Chip
+        label={tagName}
+        variant={'outlined'}
+        onDelete={() => {
+          handleDeleteBtnClick();
+        }}
+      />
+      {backdropOpen && (
+        <ConfirmCard
+          lang="en"
```
Comment (tommivk) [inline src/components/TagList/TagChip.tsx:50]:
```suggestion
          lang={lang}
```

File: src/app/api/tag/route.ts (hunk 1 above)
Comment (piamanns) [reply src/app/api/tag/route.ts]: Will fix!

File: src/components/TagList/TagChip.tsx [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+  const handleDelete = async () => {
+    const responseJson = await remove('/api/tag/', {
+      tagId: tagId,
+    });
+    notify(responseJson);
+    router.refresh();
+  };
+
+  function handleDeleteBtnClick() {
```
[the remark this answers is not in the document]
Comment (piamanns) [reply src/components/TagList/TagChip.tsx:35]: Changed this to calling setBackdropOpen directly in onDelete

File: src/app/api/tag/route.ts [hunk 4]
```diff
@@ -38,10 +38,38 @@ export async function POST(request: NextRequest) {
...
+        id: tag.tagId,
+      },
+    });
+
+    return successResponse({
+      message: t('Tags.tagDeleted'),
+      statusCode: StatusCodeType.OK,
+    });
+  } catch (error: unknown) {
+    return handleCommonErrors(error);
```
[the remark this answers is not in the document]
Comment (piamanns) [reply src/app/api/tag/route.ts:73]: Very true, will fix!

**Review by tommivk — APPROVED** [review]
Looks good!

**Pull request merged**